app/services/article_api_index_service.rb

```diff
@@ -74,18 +74,29 @@ def tag_articles
   end
 
   def top_articles
-    Article.published.order("positive_reactions_count DESC").where("published_at > ?", top.to_i.days.ago).
+    Article.published.includes(:user, :organization).
```

Comment:
we weren't loading those two relations that are accessed in the JSON rendering for the API